sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/archive/#rtirq"
+
+SRC_URI="http://www.rncbc.org/archive/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=sys-apps/util-linux-2.13"
```

Comment:
I barely believe you need this dependency explicitly here

Reply:
Okay, removed in 8e74ad02846.